### PR TITLE
enhance: add lazyload basic integration test

### PR DESCRIPTION
**Pull request by sunby**
issue: https://github.com/milvus-io/milvus/issues/31856

### CODE REVIEW

File: tests/integration/lazyload/lazyload_test.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		DbName:         dbName,
+		CollectionName: collectionName,
+	})
+	s.NoError(err)
+	err = merr.Error(status)
+	s.NoError(err)
+
+	s.WaitForLoad(ctx, collectionName)
+	req := integration.ConstructSearchRequest(dbName, collectionName, "", integration.FloatVecField, schemapb.DataType_FloatVector, []string{}, metric.L2, map[string]interface{}{}, 1, dim, 1, -1)
+	searchRes, err := s.Cluster.Proxy.Search(ctx, req)
```
Comment (jaime0815) [inline tests/integration/lazyload/lazyload_test.go:92]: how can we ensure coverage load and eviction of cache?

Comment (sunby) [reply tests/integration/lazyload/lazyload_test.go:92]: > how can we ensure coverage load and eviction of cache?

we can't. that's not the purpose of integration test. we can only ensure the functionality works fine end to end.

Comment (weiliu1031) [reply tests/integration/lazyload/lazyload_test.go:93]: single search test isn't enough, it's recommend to do more random nq test

File: tests/integration/lazyload/lazyload_test.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	fVecColumn := integration.NewFloatVectorFieldData(integration.FloatVecField, rowNum, dim)
+	hashKeys := integration.GenerateHashKeys(rowNum)
+	res, err := s.Cluster.Proxy.Insert(ctx, &milvuspb.InsertRequest{
+		DbName:         dbName,
+		CollectionName: collectionName,
+		FieldsData:     []*schemapb.FieldData{pkFieldData, fVecColumn},
+		HashKeys:       hashKeys,
+		NumRows:        uint32(rowNum),
+	})
+	s.NoError(err)
```
Comment (weiliu1031) [inline tests/integration/lazyload/lazyload_test.go:70]: need flush here to generate sealed segments, only sealed segment can verify lazy load mode

File: tests/integration/lazyload/lazyload_test.go (hunk 1 above)
Comment (sunby) [reply tests/integration/lazyload/lazyload_test.go:100]: > single search test isn't enough, it's recommend to do more random nq test

it's not stable now. I will add more later.

**Comment by jaime0815** [comment]
we can limit the disk cache size and the segment size to trigger load and evict in this test case?

**Pull request closed without merging**